docs/gen-docs/kyma_alpha_deploy.md

```diff
@@ -24,27 +24,28 @@ kyma alpha deploy [flags]
-  -c, --channel string              Select which channel to deploy from. (default "regular")
-      --dry-run                     Renders the Kubernetes manifests without actually applying them.
-  -f, --force-conflicts             Forces the patching of Kyma spec modules in case their managed field was edited by a source other than Kyma CLI.
-  -k, --kustomization stringArray   Provides one or more kustomizations to deploy. 
-                                    Each flag occurrence accepts a URL with an optional reference (commit, branch, or release) in URL@ref format or a local path to the directory of the kustomization file.
-                                    By default, Lifecycle Manager is deployed from the GitHub main branch. (default [https://github.com/kyma-project/lifecycle-manager/config/default])
-      --kyma-cr string              Provide a custom Kyma CR file for the deployment.
-      --lifecycle-manager string    Installs Lifecycle Manager with the specified image:
-                                    - Use "my-registry.org/lifecycle-manager:my-tag"" to use a custom version of Lifecycle Manager.
-                                    - Use "europe-docker.pkg.dev/kyma-project/prod/lifecycle-manager@sha256:cb74b29cfe80c639c9ee9..." to use a custom version of Lifecycle Manager with a digest.
-                                    - Specify a tag to override the default one. For example, when specifying "v20230220-7b8e9515",  the "eu.gcr.io/kyma-project/lifecycle-manager:v20230220-7b8e9515" tag is used. (default "eu.gcr.io/kyma-project/lifecycle-manager:latest")
-  -m, --module stringArray          Provide one or more modules to activate after the deployment is finished. Example: "--module name@namespace" (namespace is optional).
-  -n, --namespace string            The Namespace to deploy the the Kyma custom resource in. (default "kyma-system")
-      --open-dashboard              Opens the Busola Dashboard at startup. Only works when a graphical interface is available and when running in interactive mode
-      --templates stringArray       Provide one or more module templates to deploy.
-                                    WARNING: This is a temporary flag for development and will be removed soon. (default [https://github.com/kyma-project/kyma/modules@main])
-  -t, --timeout duration            Maximum time for the deployment. (default 20m0s)
-      --wildcard-permissions        Creates a wildcard cluster-role to allow for easy local installation permissions of Lifecycle Manager.
-                                    Allows for Lifecycle Manager usage without worrying about modules requiring specific RBAC permissions.
-                                    WARNING: DO NOT USE ON PRODUCTIVE CLUSTERS! (default true)
+  -b, --bare                          Skip applying default module templates after the deployment.
```

Comment:
I have a bit of doubt about this `bare` option, it's just for me if not reading the description, don't know what's the meaning, can we change it to something more straightforward? like `skip-templates`. Also, I think this option is not a flag that will often be used, no worth to give a short name.

Reply:
Don't really mind :) Just took that from the ACs, guess we are free in deciding that? Would rename it to `skip-default-templates` then.